content/blog/2023/06/19/2023-06-19-remove-outdated-plugins-while-using-docker.adoc

```diff
@@ -0,0 +1,339 @@
+6. Restart by hitting the `/safeRestart` endpoint.
+7. Optional: remove the plugins' remnants from the docker volume.
++
+[source,bash]
+----
+cd ~/plugins
+ls -artl *
+[...]
+-rw-r--r--   1 jenkins jenkins        8 Jun  6 13:05 bootstrap4-api.jpi.version_from_image
+-rw-r--r--   1 jenkins jenkins        0 Jun  6 13:05 bootstrap4-api.jpi.pinned
+-rw-r--r--   1 jenkins jenkins        9 Jun  6 13:05 popper-api.jpi.version_from_image
+-rw-r--r--   1 jenkins jenkins        0 Jun  6 13:05 popper-api.jpi.pinned
+drwxr-xr-x   6 jenkins jenkins     4096 Jun  6 13:05 bootstrap4-api
+drwxr-xr-x   5 jenkins jenkins     4096 Jun  6 13:05 popper-api
+[...]
+rm -fr bootstrap4-api* popper-api*
+----
+
+Well done! You have successfully eliminated any deprecated plugins (at least for now). Keep up the good work!
+
+## The intuitive (but plenty wrong) approach
+
+### You can't ignore the warning
+
+If your Jenkins instance has deprecated plugins, you'll notice a notification bell in the top right corner.
+Clicking on it will display a warning message similar to this:
+
+image::/images/post-images/2023/06/19/2023-06-19-remove-outdated-plugins-while-using-docker/the-following-plugins-are-deprecated.png[width=839,deprecated plugins warning]
+
+It suggests you to click on **Manage Jenkins** to remove the deprecated plugins.
+Once in **Manage Jenkins**, you can't (once more) ignore the warning:
+
+image::/images/post-images/2023/06/19/2023-06-19-remove-outdated-plugins-while-using-docker/the-following-plugins-are-deprecated-once-more.png[width=839,deprecated plugins warning once again]
+
+It's a clear indication that we need to take action to address this situation.
+
+### Removing deprecated plugins in the UI
+
+Follow these steps to remove the deprecated plugin:
+
+1. Click on *Plugins*, then select *Installed plugins*.
+2. In the search box, enter the name of the deprecated plugin (in this case, it's `popper`).
+
+image::/images/post-images/2023/06/19/2023-06-19-remove-outdated-plugins-while-using-docker/popper-can-t-be-uninstalled.png[width=839,search for the deprecated plugin]
+
+Unfortunately, we encounter an issue as we are unable to uninstall it.
+The checkmark is greyed out, and the *Uninstall* button (red cross) is disabled.
+Why is that?
+Some plugins have dependencies on other plugins, which in turn have dependencies on further plugins, creating a chain of dependencies.
+When you put your mouse pointer over the red cross, you'll see a tooltip that indicates the parent plugin blocking the uninstallation:
+
+image::/images/post-images/2023/06/19/2023-06-19-remove-outdated-plugins-while-using-docker/who-is-my-daddy.png[what is the parent plugin blocking the uninstallation?]
+
+In this case, `popper` is a dependency for another plugin called `bootstrap4-api`.
+Therefore, we need to remove `bootstrap4-api` first and then proceed with `popper`.
+
+Back to the [.line-through]#drawing board# search box, this time with `bootstrap4-api`.
```

Comment:
```suggestion
Back to the +++<del>+++drawing board+++</del>+++ search box, this time with `bootstrap4-api`.
```
In the deploy, it appears that the line-through is not rendering. I was searching around looking for an alternative that may work, and in my test it looks like this does the job.
<img width="1055" alt="Screenshot 2023-06-16 at 3 45 44 PM" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/65c6daa2-a2bb-4daa-a596-897a6abbaf12">